dev/TabView/TabViewItem.cpp

```diff
@@ -45,6 +45,7 @@ winrt::AutomationPeer TabViewItem::OnCreateAutomationPeer()
 void TabViewItem::OnLoaded(const winrt::IInspectable& sender, const winrt::RoutedEventArgs& args)
 {
     UpdateCloseButton();
+    OnHeaderPropertyChanged(nullptr);
```

Comment:
As a general rule, it's better to have event handlers call internal methods, and then call those internal methods when you want to have the same behavior outside of those event handlers - so, for example, you could have a method called `UpdateToolTipWithHeaderProperty()` and call that here.  Doing that adds to code readability, since the method's name now describes what we're really doing, rather than sounding like we're changing the value of the Header property.

Reply:
Will keep it in mind but since I removed this call now it's moot. :)